docs/source/tutorial/data_examples.rst

```diff
@@ -9,9 +9,15 @@ on Citrination. To access the data client, instantiate ``CitrinationClient`` and
 Uploading Files
 ---------------
 
-The ``DataClient`` class exposes a method, ``.upload`` which
-allows you to upload a file or a directory to a dataset on
-Citrination.
+The ``DataClient`` class exposes a method, ``.upload`` which allows you to
+upload a file or a directory to a dataset on Citrination using the "default"
+ingester. This method is useful for uploading JSON files that follow the PIF
+schema, as well as files that do not require additional processing with a custom
+ingester.
+
+.. attention::
+  For files that require additional processing, see the documentation for the
+  ``upload_with_ingester`` and ``upload_with_template_csv_ingester`` sections.
 
 This method is parameterized with the following values:
```

Comment:
In a normal workflow is the `dataset_id` argument something they will have to write code to obtain? If it is - you might want to add a note with a link to the docs that explain how to get that ID

Reply:
The `dataset_id` argument is just an integer/string, you kind of just need to know which dataset you want to work with. You can't really obtain it from code unless you've just created a new dataset.